Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -109,17 +109,18 @@ def fetch_indicators(self, limit: Optional[int], type: list = None, malicious_co
         Returns:
             (list): parsed indicators
         """
+        filter = f'({filter})' if filter else ''
         if type:
             type_fql = self.build_type_fql(type)
-            filter = f'{type_fql}+{filter}' if filter else type_fql
+            filter = f'({type_fql})+{filter}' if filter else f'({type_fql})'
 
         if malicious_confidence:
-            malicious_confidence_fql = f"malicious_confidence:'{malicious_confidence}'"
-            filter = f"{filter}+{malicious_confidence_fql}" if filter else malicious_confidence_fql
+            malicious_confidence_fql = ','.join([f"malicious_confidence:'{item}'" for item in malicious_confidence])
```

Comment:
Does this mean we were able to make the query of the filter by a list of values work?

Reply:
yes